SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.7 BLOCK -->
```

Comment:
We shouldn't have this file in the GSF public repo as it relates to the official Microsoft repo.